CHANGELOG.md

```diff
@@ -20,10 +19,13 @@ All notable changes for each version of this project will be documented in this
         - The footerTemplate property is removed.
     - `igxColumnGroup`
         - The footerTemplate property is removed.
-    - `igxCombo`
-        - **Breaking change** igxCombo default `width` is set to 100%
-        - **Breaking change** `itemsMaxWidth` is renamed to `itemsWidth`
-
+    - exposed `autosize()` method on IgxColumnComponent. It allows the user to programatically change the size of a column according to it's largest visible cell.
```

Comment:
We might put `IgxColumnComponent` in quotes. Also should it start with a small or a capital letter?